core/src/main/java/org/apache/carbondata/core/util/SessionParams.java

```diff
@@ -229,6 +229,12 @@ private boolean validateKeyValue(String key, String value) throws InvalidConfigu
           if (!isValid) {
             throw new InvalidConfigurationException("Invalid value " + value + " for key " + key);
           }
+        } else if (key.startsWith(CarbonLoadOptionConstants.CARBON_TABLE_LOAD_SORT_SCOPE)) {
+          isValid = CarbonUtil.isValidSortOption(value);
+          if (!isValid) {
+            throw new InvalidConfigurationException("The sort scope " + key
+                + " can have only either BATCH_SORT or LOCAL_SORT or NO_SORT.");
```

Comment:
Sort scope can be global as well. Add in exception message

Reply:
Done.